cogs/moderation.py

```
@@ -0,0 +1,258 @@
+    async def ban(self, ctx, member: discord.Member, *, reason: str = "No reason provided"):
+        """Ban a member from the server"""
+        if member.top_role >= ctx.author.top_role and ctx.author != ctx.guild.owner:
+            return await ctx.send("❌ You cannot ban someone with a higher or equal role!")
+        
+        if member.top_role >= ctx.guild.me.top_role:
+            return await ctx.send("❌ I cannot ban someone with a higher or equal role!")
+        
+        try:
+            await member.send(f"You have been banned from {ctx.guild.name}. Reason: {reason}")
+        except:
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)